logger.go

```diff
@@ -0,0 +1,14 @@
+package gocql
+
+import (
+	"io/ioutil"
+	"log"
+)
+
+type StdLogger interface {
+	Print(v ...interface{})
+	Printf(format string, v ...interface{})
+	Println(v ...interface{})
+}
+
+var Logger StdLogger = log.New(ioutil.Discard, "", log.LstdFlags)
```

Comment:
I feel this should be the default go logger and users can silence it in case of need to